package.json

```diff
@@ -13,9 +13,10 @@
   "author": "Julie Ralph <ju.ralph@gmail.com>",
   "dependencies": {
     "adm-zip": "0.4.7",
+    "chalk": "^1.1.3",
```

Comment:
Added a try catch for chalk. If chalk is not available, it will not use chalk logging features.